__tests__/workers/cdc.ts

```diff
@@ -975,3 +1005,54 @@ describe('reputation event', () => {
     expect(user.reputation).toEqual(5);
   });
 });
+
+describe('submission', () => {
+  type ObjectType = Submission;
+  const id = randomUUID();
+  const base: ChangeObject<ObjectType> = {
+    id,
+    userId: '1',
+    url: '',
+    reason: '',
+    status: SubmissionStatus.NotStarted,
+    createdAt: Date.now(),
+  };
+
+  it('should notify crawler for this article', async () => {
+    const after: ChangeObject<ObjectType> = base;
+    await expectSuccessfulBackground(
+      worker,
+      mockChangeMessage<ObjectType>({
+        after,
+        before: null,
+        op: 'c',
+        table: 'submission',
+      }),
+    );
+    expect(notifySubmissionCreated).toBeCalledTimes(1);
+    expect(mocked(notifySubmissionCreated).mock.calls[0].slice(1)).toEqual([
+      { url: after.url, submissionId: after.id, sourceId: 'TBD' },
```

Comment:
`sourceId` should be fixed because we agreed to add the migration with the new source